envision/web/src/index.html

```diff
@@ -7,6 +7,9 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <title>Envision</title>
+
+    <!-- link this EBML.js to fix video seeking issues -->
+    <script src="https://www.webrtc-experiment.com/EBML.js"></script>
```

Comment:
The tricky thing is using the `getSeekableBlob` required to link this EBML.js file. I didn't see any documentation mentioning this but I saw an error in the console log.

Reply:
It mentions on line 1904-1907 in https://recordrtc.org/RecordRTC.js.html. 
"EBML.js copyrights goes to: https://github.com/legokichi/ts-ebml"

Reply:
It mentions on line 1904-1907 of https://recordrtc.org/RecordRTC.js.html
"EBML.js copyrights goes to: https://github.com/legokichi/ts-ebml"

Reply:
It looks OK, they have MIT license.